### ISSUE TITLE
Enable line comments when whitespace changes are suppressed on pull requests

### ISSUE THREAD

**Issue by simonbingham** [isaacs/github#394]
Currently, when whitespace changes are suppressed on pull requests (by appending `?w=1` to the url) it's not possible to add line comments. It would be great if it was possible to do so.

**Comment by cirosantilli** [comment]
Please link to a live example on a minimal repo whenever possible.

**Comment by mwarkentin** [comment]
Yes.. this drives me nuts.

Example: https://github.com/mwarkentin/django-watchman/pull/74/files?w=1

**Comment by berdandy** [comment]
Yes, this is something that I would also love to see. It makes it challenging to review a PR that contains, for instance, line-ending changes when you're dealing with disparate operating systems.

**Comment by efgerrol** [comment]
i just ran into that issue today.

**Comment by photusenigma** [comment]
This has recently become one of the things I'm wishing for for Christmahanakwanzika.

**Comment by pgacv2** [comment]
+1 from my team
And apparently +1 from a lot of other users, as well: https://github.com/holman/ama/issues/465
